sql/report_queries/acrl/collections_expenditures/ACRL Collection Expenditures A.sql

```diff
@@ -0,0 +1,207 @@
+    inv.approval_date::date AS inv_approval_date,
+    COALESCE(invl.sub_total,0) AS invl_sub_total,
+    ila.adjustment_value AS invl_adjustment_value,
+    ila.adjustment_description AS invl_adj_desc,
+    coalesce (invl.total, 0)::numeric(12, 2) AS invl_value,
+    coalesce (invadjext.invls_total, 0) ::numeric(12, 2) AS invls_total,-- This is coming from the invoice_adjustments_ext table
+    inv.exchange_rate::NUMERIC(12,3) AS inv_exchange_rate,
+    inv.currency AS inv_currency,
+    ft.transaction_type AS transaction_type,
+    COALESCE(invadjext.ratio_of_inv_adj_per_invoice_line,0) * coalesce(ita.transaction_amount,0)::numeric(12,2) AS transaction_inv_adj_dist_converted,
+    COALESCE(ilta.transaction_total_per_invl,0) AS transaction_amount_per_invl_converted,
+   	COALESCE(ilta.transaction_total_per_invl,0) + (COALESCE(invadjext.ratio_of_inv_adj_per_invoice_line,0) * COALESCE(ita.transaction_amount,0)) AS total_paid_converted
+       
+FROM
+	invoice_invoices AS inv
+    LEFT JOIN invoice_lines AS invl ON invl.invoice_id = inv.id 
+    LEFT JOIN po_lines AS pol ON pol.id = invl.po_line_id
+    LEFT JOIN po_purchase_orders AS po ON pol.purchase_order_id = po.id
+    LEFT JOIN folio_reporting.invoice_lines_adjustments AS ila ON ila.invoice_line_id = invl.id
+    LEFT JOIN folio_reporting.instance_ext AS iext ON iext.instance_id = pol.instance_id
+    LEFT JOIN folio_reporting.instance_formats AS ifo ON ifo.instance_id = iext.instance_id
+    LEFT JOIN folio_reporting.invoice_adjustments_ext AS invadjext ON invadjext.invl_id = invl.id
+    LEFT JOIN folio_reporting.po_lines_phys_mat_type AS pol_phys_type ON pol.id = pol_phys_type.pol_id
+	LEFT JOIN folio_reporting.po_lines_er_mat_type AS pol_er_type ON pol.id = pol_er_type.pol_id
+	LEFT JOIN finance_transactions AS ft ON ft.source_invoice_line_id = invl.id
+	LEFT JOIN invl_transac_amount_total AS ilta ON ilta.invl_id = invl.id
+	LEFT JOIN inv_transac_amount_total AS ita ON ita.transaction_source_invoice_id = inv.id
+		
+
+WHERE
+	invl.invoice_line_status LIKE 'Paid'
+	AND (inv.payment_date::date >= (SELECT payment_date_start_date FROM parameters))
+	AND (inv.payment_date::date < (SELECT payment_date_end_date FROM parameters))
+	AND (inv.payment_date >= (SELECT payment_date_start_date FROM parameters))
+	AND (inv.payment_date < (SELECT payment_date_end_date FROM parameters))
+	AND (po.order_type = (SELECT order_type FROM parameters) OR (SELECT order_type FROM parameters) = '')
+	AND (pol.order_format = (SELECT order_format FROM parameters) OR (SELECT order_format FROM parameters) = '')
+	AND (ifo.format_name = (SELECT instance_format_name FROM parameters) OR (SELECT instance_format_name FROM parameters) = '')
+	AND (iext.mode_of_issuance_name = (SELECT instance_mode_of_issuance FROM parameters) OR (SELECT instance_mode_of_issuance FROM parameters) = '')
+	AND (ft.transaction_type = (SELECT transaction_type FROM parameters) OR (SELECT transaction_type FROM parameters) = '')
+
+GROUP BY 
```

Comment:
It looks like the formatting is a little off. Generally we indent with the Group by. There were also some extra lines here and there. PG Format is used (https://github.com/folio-org/folio-analytics/blob/main/sql/pg_format.conf). Though I've also asked Nassib in the past if he could run the query through this command line tool.

Reply:
I can change that, no problem.

Reply:
It's been updated.